packages/demo/package.json

```
@@ -35,12 +36,12 @@
     "@web3-onboard/ledger": "^2.3.2",
     "@web3-onboard/magic": "^2.1.3",
     "@web3-onboard/portis": "^2.1.3",
-    "@web3-onboard/sequence": "^2.0.4",
-    "@web3-onboard/torus": "^2.2.0-alpha.1",
-    "@web3-onboard/trezor": "^2.3.2",
     "@web3-onboard/tallyho": "^2.0.1",
-    "@web3-onboard/web3auth": "^2.1.4-alpha.1",
-    "@web3-onboard/walletconnect": "^2.2.2-alpha.1",
+    "@web3-onboard/torus": "^2.1.3",
```

This looks to be overriding the current alpha packages in a few spots
```suggestion
    "@web3-onboard/torus": "^2.2.0-alpha.1",
```